File: Problem007/java/Main.java
//104743
class Main{
	public static void main(String[] args){
		int i = 0;
		int n = 1;
		while(i < 10001){
			n++;
			if(isPrime(n)){
				i++;
			}
		}
		System.out.println(n);
	}
	
	static boolean isPrime(int n){
		if(n < 2)return false;
		for(int i=2;i*i<=n;i++){
			if(n % i == 0)return false;
		}
		return true;
	}
}

File: Problem007/go/main.go
package main

import (
	"fmt"
)

func isPrime(n int) bool {
	if n < 2 {
		return false
	}
	for i := 2; i*i <= n; i++ {
		if n%i == 0 {
			return false
		}
	}
	return true
}

func main() {
	i := 1
	n := 0
	for n < 10001 {
		i++
		if isPrime(i) {
			n++
		}
	}
	fmt.Println(i)
}
